config/user.js

```diff
@@ -1,20 +1,6 @@
 const CERTIFICATE_DEVSERIAL = '00';
 export default {
-  secret: process.env.NODE_ENV === 'test' ? 'secret for tests' : process.env.AUTH_SECRET,
   auth: {
-    access: {
-      serial: {
-        enabled: false
-      },
-      session: {
-        enabled: true
-      },
-      jwt: {
-        enabled: true,
-        tokenExpiresIn: '1m',
-        refreshTokenExpiresIn: '7d'
-      }
-    },
     password: {
```

Comment:
Why do we leave auth configs for `user` and `certificate` authentication here?

Reply:
because all work with these settings occurs only in the user module